Sources/Core/Pretty.swift

```diff
@@ -9,7 +9,8 @@
 import Foundation
 
 struct Pretty {
-    let formatter: PrettyFormatter
+    var formatter: PrettyFormatter
+    var timeZone: TimeZone = .current
```

Comment:
Use system timeZone in default for print `Date` type.
This is useful than `print()` that use GMT.

```swift
let formatter = DateFormatter()
formatter.dateFormat = "yyyy-MM-dd HH:mm ZZZZZ"
formatter.timeZone = TimeZone(identifier: "Asia/Tokyo")

let date = formatter.date(from: "2020-03-24 10:00 +0900")!
print(date)      // => 2020-03-24 01:00:00 +0000
debugPrint(date) // => 2020-03-24 01:00:00 +0000

Debug.p  >>> date // 2020-03-24 10:00:00
Debug.pd >>> date // Date("2020-03-24 10:00:00 +09:00")
```